### DISCUSSION TITLE
请问访问控制中如何设置指定ip通过代理

### DISCUSSION THREAD

**Discussion by orthwang** [Openwrt-Passwall/openwrt-passwall2#123] [Q&A]
研究了一下午 还是不懂如何利用访问端口和路由规则控制  进行访问控制  让局域网中的指定ip通过代理

请大神指导

**Comment by xlyeah** [comment]
我是在分流设置里 新建一个项 然后设置局域网ip   底下域名和ip地址设置需要代理的域名和地址，然后分流的时候，在这个规则底下选择代理节点 或者直连 就行了

**Reply by xlyeah** [reply]
https://github.com/xiaorouji/openwrt-passwall2/discussions/103 其实你按照这种方法也行 反向操作

**Reply by orthwang** [reply]
万分感谢

**Comment by caopeng19911002** [comment]
域名怎么设置啊，我设置了ip好像不能生效

**Comment by handsomeji** [comment]
最新的源码已经可以单独开启访问控制
